homeassistant/components/binary_sensor/owlet.py

```diff
@@ -0,0 +1,84 @@
+"""
+Support for Owlet binary sensors.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/binary_sensor.owlet/
+"""
+from datetime import timedelta
+
+from homeassistant.components.binary_sensor import BinarySensorDevice
+from homeassistant.components.owlet import DOMAIN
```

Comment:
Because this platform runs under the `binary_sensor` domain, it should alias the `DOMAIN` from `owlet`.

`from homeassistant.components.owlet import DOMAIN as OWLET_DOMAIN`